index.html

```diff
@@ -224,8 +224,8 @@ <h3 id="klonk">Klonk</h3>
 					<h2>##books irc channel</h2>
 						<p>Our irc channel resides on freenode, we are a group of avid book readers and just like to join and talk about what we are reading, or whatever fancy our minds.</p>
 						<p>We started our goodreads group as a way of having a centralized place where to share what we read in general, you can see the group here <a href="https://www.goodreads.com/topic/show/18178432-restarting-the-club-with-some-good-new-scifi">#books goodread group</a></p>
-						<p>You are always welcome to join us and talk. If you know how to use an irc client, just join irc://irc.freenode.net/##books</p>
-						<p>If not just join us at <a href="http://webchat.freenode.net/?nick=bookreader..&channels=%23%23books">freenode webchat</a></p>
```

Comment:
How important is the nick bookreader? I could restore it, I just didn't notice it before making a PR.